app/validation.py

```diff
@@ -60,11 +61,12 @@ def get_validator(schema_name, enforce_required=True, required_fields=None):
     if enforce_required:
         schema = _SCHEMAS[schema_name]
     else:
-        schema = _SCHEMAS[schema_name].copy()
+        schema = copy.deepcopy(_SCHEMAS[schema_name])
```

Comment:
I'm guessing this was left from the `anyOf` modification commit. I think shallow `.copy()` should be still ok here.